docker/main/Dockerfile

```diff
@@ -318,6 +318,7 @@ FROM --platform=$BUILDPLATFORM node:20 AS web-build
 
 WORKDIR /work
 COPY web/package.json web/package-lock.json ./
+
```

Comment:
Could you remove line spacing changes from files that don't have any code modifications?